arches_lingo/src/arches_lingo/components/concept/ConceptImages/ConceptImages.vue

```diff
@@ -0,0 +1,82 @@
+<script setup lang="ts">
+import { onMounted, ref } from "vue";
+
+import ProgressSpinner from "primevue/progressspinner";
+
+//import ConceptImagesEditor from "@/arches_lingo/components/concept/ConceptImages/components/ConceptImagesEditor.vue";
+import ConceptImagesViewer from "@/arches_lingo/components/concept/ConceptImages/components/ConceptImagesViewer.vue";
+
+import { EDIT, VIEW } from "@/arches_lingo/constants.ts";
+
+import { fetchLingoResourcePartial } from "@/arches_lingo/api.ts";
+
+import type { ConceptImages, DataComponentMode } from "@/arches_lingo/types.ts";
+
+const props = defineProps<{
+    mode: DataComponentMode;
+    sectionTitle: string;
+    componentName: string;
+    graphSlug: string;
+    nodegroupAlias: string;
+    resourceInstanceId: string | undefined;
+    tileId?: string;
+}>();
+
+const isLoading = ref(true);
+const tileData = ref<ConceptImages>();
+
+const shouldCreateNewTile = Boolean(props.mode === EDIT && !props.tileId);
+
+onMounted(async () => {
+    if (
+        props.resourceInstanceId &&
+        (props.mode === VIEW || !shouldCreateNewTile)
+    ) {
+        const sectionValue = await getSectionValue();
+        tileData.value = sectionValue.aliased_data[props.nodegroupAlias];
+    }
+
+    isLoading.value = false;
+});
+
+async function getSectionValue() {
+    try {
+        return await fetchLingoResourcePartial(
+            props.graphSlug,
+            props.resourceInstanceId as string,
+            props.nodegroupAlias,
+        );
+    } catch (error) {
+        console.error(error);
+    }
+}
+</script>
+
+<template>
+    <ProgressSpinner
+        v-if="isLoading"
+        style="width: 100%"
+    />
+
```

Comment:
nit: Add an error message component. I think that's been added into sections after you've pushed this out for review so just getting it in-line with the rest 👍